src/transformers/pipelines/__init__.py

```diff
@@ -387,8 +387,11 @@
     if values["type"] == "text":
         NO_FEATURE_EXTRACTOR_TASKS.add(task)
         NO_IMAGE_PROCESSOR_TASKS.add(task)
-    elif values["type"] in {"audio", "image", "video"}:
+    elif values["type"] in {"image", "video"}:
         NO_TOKENIZER_TASKS.add(task)
+    elif values["type"] in {"audio"}:
+        NO_TOKENIZER_TASKS.add(task)
+        NO_IMAGE_PROCESSOR_TASKS.add(task)
```

Comment:
to avoid `AutomaticSpeechRecognitionPipeline` to load an image processor (with the new addition below in this file)